src/helpers/virtualKeyboard.ts

```diff
@@ -8,7 +8,7 @@ function currentElement() {
 }
 
 function searchInput() {
-  return document.querySelector('.searchInput') as HTMLInputElement
+  return document.querySelector('#search') as HTMLInputElement
```

Comment:
Maybe add a comment here as well so we don't change this in the future and break it again.